lib/ai_helper/assistant.rb

```diff
@@ -27,9 +27,19 @@ def available_prompts
 
             prompts =
               prompts.map do |prompt|
-                translation =
-                  I18n.t("discourse_ai.ai_helper.prompts.#{prompt.name}", default: nil) ||
-                    prompt.translated_name || prompt.name
+                if prompt.name == "translate"
+                  locale = user.effective_locale
+                  locale_hash = LocaleSiteSetting.language_names[locale]
+                  translation =
+                    I18n.t(
+                      "discourse_ai.ai_helper.prompts.#{prompt.name}",
```

Comment:
You could even use the actual translation key in this case.

```suggestion
                      "discourse_ai.ai_helper.prompts.translate",
```